assets/src/edit-story/panels/panel/content.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { useContext } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import panelContext from './context';
+
+const Form = styled.form`
+	margin: 10px 20px;
+	overflow: auto;
+`;
+
+function Content( { children, ...rest } ) {
+	const { state: { isCollapsed, height } } = useContext( panelContext );
+
+	if ( isCollapsed ) {
+		return null;
+	}
+
+	const formStyle = {
+		height: height === null ? 'auto' : `${ height }px`,
+	};
+
+	return (
+		<Form style={ formStyle } { ...rest }>
```

Comment:
Do we need a `role="region"` here?

Reply:
The [WAI-ARIA docs](https://www.w3.org/TR/wai-aria-practices-1.1/#wai-aria-roles-states-and-properties) say:

> Avoid using the `region` role in circumstances that create landmark region proliferation, e.g., in an accordion that contains more than approximately 6 panels that can be expanded at the same time.

I think we'll easily reach that with the editor, so I'd vote no on using it.

Reply:
Fair enough. Agreed